lib/global.js

```diff
@@ -1,14 +1,3 @@
-var createTorrent = require('create-torrent')
-var wrtc = require('wrtc')
+var wrtc = require('electron-webrtc')()
 
 global.WRTC = wrtc
-
-global.WEBTORRENT_ANNOUNCE = createTorrent.announceList
```

Comment:
I guess it might make sense to check if there is webrtc support and in that case to add these torrents, in webtorrent itself.
That would make it possible to remove all these adding of trackers in these projects.
> still handles this restriction by themself

Really, it's not a restriction. If you don't define WEBTORRENT_ANNOUNCE , it doesn't add any trackers at all.

So yeah I think either revert the removal here, or add it in webtorrent itself (or both until it's merged and published in webtorrent).